neon_utils/default_configurations/default_user_conf.yml

```diff
@@ -48,3 +48,8 @@ location:
   country: United States
   tz: America/Los_Angeles
   utc: '-8.0'
+
+speed_mode:
+  mode: quick
+  available_modes: [quick, thoughtful]
+  mode_extension_time: {quick: 1, thoughtful: 10}
```

Comment:
I'm wondering if this belongs in config or somewhere in code.. Main rationale for taking it out of config is that we can't change this on existing installations later, so we won't be able to increase/decrease default waits or add others